src/containers/Messages/Message.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+const OpenAPISampler = require('openapi-sampler');
+
+import { Header, H2, H3, H4, Markdown, HeaderParagraph, TableColumn, Tag, DeprecatedBadge } from '../../components';
+
+import { Map, Message, Schema } from '../../common';
+
+import { 
+  Message as MessageWrapper,
+  MessageHeader,
+  MessageHeaders,
+  MessageHeadersHeader,
+  MessagePayload,
+  MessagePayloadHeader,
+  MessageTags,
+  MessageTagsHeader,
+ } from './styled';
+
+import SchemaComponent from '../Schemas/Schema';
+
+export interface MessagesProps {
+  title?: string,
+  message: Message,
+  hideTags?: boolean,
+}
+
+class MessageComponent extends Component<MessagesProps> {
+  public render() {
+    const { title, message, hideTags } = this.props;
+
+    const headers = message.headers;
+    const payload = message.payload;
+
+    return (
+      message ?
+        <MessageWrapper>
+          <MessageHeader>
+            {title ? <H3>{title} {message.deprecated ? <DeprecatedBadge>Deprecated</DeprecatedBadge> : null}</H3> : null}
```

Comment:
Please avoid ternary operator inside ternary operator- but it will be resolved once replacing first ternary operator with guard